src/general-redaction/rapidjsonRedactor.cpp

```diff
@@ -77,7 +51,12 @@ bool RapidjsonRedactor::redactMemberByPath(rapidjson::Value &value, std::string
                         return true;
                     }
                     else if (type == "String" && target == "transmission") {
-                        value["transmission"] = "UNAVAILABLE";
+                        value["transmission"] = "unavailable";
```

Comment:
This was changed to lower case to match how the OdeBsmJson output writes it. This is how the J2735 defines it as well.

Reply:
Addressed.